### DISCUSSION TITLE
Log file? Troubleshooting downloaded vs available videos

### DISCUSSION THREAD

**Discussion by cdvatx** [Dineshkarthik/telegram_media_downloader#344] [Q&A]
In the channel I'm downloading, there are 214 videos showing up when counting their video media.
Only about 120 or so are downloaded.

How do I go about troubleshooting what's going?

**Comment by cdvatx** [comment]
Anyone?

**Reply by tangyoha** [reply]
@cdvatx,you can judge the file size, if file size equal zero,throw exception and record the message id to `ids_to_retry`.
